src/components/Coaching/CoachingList.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Divider, styled, Typography } from '@material-ui/core';
+import { EcoOutlined } from '@material-ui/icons';
+import { Skeleton } from '@material-ui/lab';
+import React, { ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+import { CoachingRow } from './CoachingRow/CoachingRow';
+import {
+  CoachFragment,
+  useLoadCoachingListQuery,
+} from './LoadCoachingList.generated';
+
+const CoachingListWrapper = styled(Box)(({ theme }) => ({
+  width: '100%',
+  padding: theme.spacing(1),
+}));
+const CoachingTitleWrapper = styled(Box)(({ theme }) => ({
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'row',
```

Comment:
Shouldn't need this as the direction's initial value should be ```row``` by default.

Reply:
@gyasistory You don't need to style Box component.
You can style it using only the apis.
```
<Box display="flex" width="100%">
...
</Box>
```